### PR TITLE
Update ChannelSftp.java

### PR DESCRIPTION
**Pull request by mironabr**
flush

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.github.mwiede</groupId>
-    <artifactId>jsch</artifactId>
+    <artifactId>k2-jsch</artifactId>
```
Comment (mwiede) [inline pom.xml:5]: I think this was added by mistake right?

**Comment by mwiede** [comment]
@mironabr as said in #411 can we have a failing integration test which this PR would solve?

File: pom.xml (hunk 1 above)
Comment (mironabr) [reply pom.xml]: Right. Currently we are using our fork in our code.

**Comment by mironabr** [comment]
> @mironabr as said in #411 can we have a failing integration test which this PR would solve?

It is difficult to do that.  The issue does not happen any time. I does not happen on any SFTP server. I guess it depend on the SFTP server type/configuration. It happen on our local test servers. I saw lot of talking about it over the internet

**Comment by mwiede** [comment]
we took the change in [86bc2e6](https://github.com/mwiede/jsch/pull/431/commits/86bc2e6ef86d9bc8137887717149e7fa80d2cb22) so I will close this PR

**Pull request closed without merging**